oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollectorIT.java

```diff
@@ -2290,9 +2286,6 @@ public void testBundlingAndLatestSplit() throws Exception {
 
     @Test
     public void testBundledPropUnmergedBCGC() throws Exception {
-        assumeTrue(fullGcMode != FullGCMode.ORPHANS_EMPTYPROPS_KEEP_ONE_ALL_PROPS);
```

Comment:
this was introduced due to [this flaky test failure](https://issues.apache.org/jira/browse/OAK-10843?focusedCommentId=17850687&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17850687) which happened on mongo - and IIUC then there's no fix for this. So the flakyness is likely still to be there. What's the idea of still enabling it?

Reply:
I will revert the changes. I don't have a fix